### PR TITLE
Made README easier to follow for locally setting up Fabrik.

### PR DESCRIPTION
**Pull request by haabhi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,13 +25,13 @@ Setting up Fabrik on your local machine is really easy. You can setup Fabrik usi
 
 2. Rename `settings/dev.sample.py` as `dev.py`.
 
-    ```
+    ```shell
     cp settings/dev.sample.py settings/dev.py
```
Comment (Ram81) [inline README.md]: remove all added `shell` , `python`, `sql`. It doesn't affect the markdown.

File: README.md [hunk 2]
```diff
@@ -144,20 +158,19 @@ Setting up Fabrik on your local machine is really easy. You can setup Fabrik usi
...
+```shell
 npm install
 npm install --save-dev json-loader
 sudo npm install -g webpack
 webpack --progress --watch --colors
 ```
 
-9. To start celery worker
+9. Start celery worker
 ```
```
Comment (Ram81) [inline README.md]: Can you mention here that you need celery worker needs to be executed parallely in a separate terminal

File: README.md [hunk 3]
```diff
@@ -144,20 +158,19 @@ Setting up Fabrik on your local machine is really easy. You can setup Fabrik usi
...
     python manage.py migrate
     ```
 8. Install node modules
-```
+```shell
 npm install
 npm install --save-dev json-loader
 sudo npm install -g webpack
 webpack --progress --watch --colors
 ```
```
Comment (Ram81) [inline README.md]: Can you mention here that you need webpack needs to be executed parallely in a separate terminal to provide hot-reload feature i.e. changes to UI code getting reflected in real time.

**Comment by haabhi** [comment]
How does it look now?

File: README.md [hunk 4]
```diff
@@ -145,23 +158,30 @@ Setting up Fabrik on your local machine is really easy. You can setup Fabrik usi
...
-```
-
-9. To start celery worker
-```
-celery -A ide worker --app=ide.celery_app  --loglevel=info
-```
+    ```
+    npm install
+    npm install --save-dev json-loader
+    sudo npm install -g webpack
```
Comment (thatbrguy) [inline README.md]:
```suggestion
    sudo npm install -g webpack@1.15.0
```
Minor suggestion. People usually have issues with webpack version, and we found the above version to work for everyone. Can you add this change?

Documentation looks neat though! Good job.

**Comment by thatbrguy** [comment]
@haabhi Can you just make that minor edit involving webpack? Otherwise LGTM :+1: @Ram81

**Comment by haabhi** [comment]
@thatbrguy Added the edit

File: README.md [hunk 5]
```diff
@@ -69,36 +69,49 @@ Setting up Fabrik on your local machine is really easy. You can setup Fabrik usi
...
+    git clone --recursive https://github.com/Cloud-CV/Fabrik.git && cd Fabrik
     ```
 
 3. Rename settings/dev.sample.py as settings/dev.py and change credential in settings/dev.py
     ```
     cp settings/dev.sample.py settings/dev.py
     ```
-    Replace the hostname to ``` localhost ``` in settings/dev.py line 14.
+    * Replace the hostname to ``` localhost ``` in settings/dev.py line 14.
+    It should now look like this:
```
Comment (Ram81) [inline README.md]: `line 14` change this to `line 15`

Comment (haabhi) [reply README.md]: Done :+1:

Comment (Ram81) [reply README.md]: Also add a new line. It doesn't look good in markdown

File: README.md [hunk 6]
```diff
@@ -69,36 +69,49 @@ Setting up Fabrik on your local machine is really easy. You can setup Fabrik usi
...
 
-4. Install redis server and replace the hostname to ``` localhost ``` in settings/common.py line 99.
+4. Install redis server  
     ```
     sudo apt-get install redis-server
     ```
-    Replace celery result backend in settings/common.py line 122 with
+    * Replace the hostname to ``` localhost ``` in settings/common.py line 115.
+
+    ```
```
Comment (Ram81) [inline README.md]: Also add a new line. It doesn't look good in markdown

File: README.md [hunk 7]
```diff
@@ -69,36 +69,49 @@ Setting up Fabrik on your local machine is really easy. You can setup Fabrik usi
...
     ```
-    CELERY_RESULT_BACKEND = 'redis://redis:6379/0'
+
+    * Replace celery result backend in settings/common.py line 122 with localhost.
+    ```
+    CELERY_RESULT_BACKEND = 'redis://localhost:6379/0'
     ```
-    Replace celery broker url and result backend hostname to ``` localhost ``` in ide/celery_app.py line 8 with
+    * Replace celery broker url and result backend hostname to ``` localhost ``` in ide/celery_app.py, line 8.
     ```
```
Comment (Ram81) [inline README.md]: Also add a new line. It doesn't look good in markdown

File: README.md [hunk 8]
```diff
@@ -69,36 +69,49 @@ Setting up Fabrik on your local machine is really easy. You can setup Fabrik usi
...
-    broker='redis://redis:6379/0'
-    backend='redis://redis:6379/0'
+    app = Celery('app', broker='redis://localhost:6379/0', backend='redis://localhost:6379/0', include=['ide.tasks'])
     ```
```
Comment (Ram81) [inline README.md]: Also add a new line. It doesn't look good in markdown

Comment (haabhi) [reply README.md]: Should I add new lines before every code spans throughout the markdown? It will look more readable that way.

**Comment by Ram81** [comment]
@haabhi LGTM 👍 , once done with minor changes let me know.

**Comment by haabhi** [comment]
@Ram81 I added new lines before and after all code blocks and fixed some indents in lists.

**Pull request merged**